andela/travel_tool_front
# #160011600 Change the default favicon

#### What does this PR do?
This PR replaces the default react app favicon with one for Travela

#### Description of Task to be completed?
- Add the favicon image to the front-end of Travela

#### How should this be manually tested?
- Clone the repo `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory 
- Checkout this branch with `git checkout ch-change-favicon-160011600`
- Install dependencies with `yarn install`
- Ensure docker is running and start the server with `make start`
- Check the browser tab where the application is opened and you should see the Travela logo as the favicon

#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#160011600

#### Screenshots (if appropriate)
#### Questions:
